graphene_pydantic/converters.py

```diff
@@ -130,12 +130,12 @@ def convert_pydantic_field(
     #   from the field's base model
     # - maybe even (Sphinx-style) parse attribute documentation
     field_kwargs.setdefault("description", field.field_info.description)
+    
+    field_type = field_kwargs.pop("type")
```

Comment:
😞 Unfortunately, this throws a `KeyError` with versions of `graphene` that don't provide `type`. I suggest updating this code to look like this:
```python
# Handle Graphene 2 and 3
field_type = field_kwargs.pop("type", field_kwargs.pop("type_", None))
if field_type is None:
    raise ValueError("No field type could be determined.")
```

It would also be great if you could update the `tox` tests to run against Graphene 2 as well.